src/qibo/quantum_info/basis.py

```diff
@@ -145,17 +152,22 @@ def pauli_basis(nqubits: int, normalize: bool = False, vectorize: bool = False):
             f"vectorize must be type bool, but it is type {type(vectorize)} instead.",
         )
 
-    basis = [matrices.I, matrices.X, matrices.Y, matrices.Z]
+    order = None if not vectorize else order
```

Comment:
I already wrote that in the documentation. If it is necessary, I'd prefer to set order to `None` by default since the default is `vectorize=False` and raise an error if the user sets `vectorize=True` without specifying `order`.

Reply:
> I already wrote that in the documentation.

I also saw the docstring, but usually if you are changing something it is preferable to raise warning.

> If it is necessary, I'd prefer to set order to `None` by default since the default is `vectorize=False` and raise an error if the user sets `vectorize=True` without specifying `order`.

I completely agree.



I agree with setting order to `None` if  `vectorize` is

Reply:
Done